### DISCUSSION TITLE
What to do when error exceeds estimate?

### DISCUSSION THREAD

**Discussion by emilyjunkins** [adw96/breakaway#100] [Q&A]
Hi folks,

How do you handle large CI spreads for samples that have error close to or exceeding the estimate? After running `breakaway` and `breakaway_nof1`, samples with higher estimates result in higher error and very large upper CI that are nearly un-plottable without transforming the Y axis (log10).

Does this indicate something about the data (other than an inability to estimate richness)?

Is transforming the axis appropriate for plotting?

**Comment by adw96** [comment]
Another great question! You have correctly interpreted what this says -- this indicates no precision in estimating richness. However, I don't believe that this is anything else about that sample. 

While I know that this is annoying, personally I would much rather have an estimator that says "nope, I don't know what's going on here" rather than that incorrectly pretends it can estimate richness! 

Regarding transforming axes for plotting -- sure, go for it! However, I would probably just let the CI's extend beyond the plotted y-axis range. (I can't remember how to do this off the top of my head, but it is possible with both `plot` and `ggplot`.)

Hope this helps and thanks again for the question!
